src/Components/NavBar/NavbarItem/component.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+import Nav from 'react-bootstrap/Nav';
+
+type ItemProps = {
+    linkTo: String,
+    label: String,
+    disabled: boolean,
+    element: node
+};
+
+class NavBarItem extends Component<ItemProps> {
+    render(): Node {
+        return (
+            <Nav.Item>
+                <Nav.Link
+                    as={this.props.element}
+                    disabled={this.props.disabled}
+                    href={'/' + this.props.linkTo}
```

Comment:
Instead of custom handler for each prop here, you should create `react-components-of-trust` component for Nav.Link and use it here.

Nav.Link component can simply expand props <Nav.Link.Of.Trust {...this.props}>

Reply:
And let who ever is going to use `react-components-of-trust` to build `Nav.Link` the way they need.

Reply:
oh okay so basically Link component should be used here instead of Nav.Link